primitives/runtime/src/generic/digest.rs

```diff
@@ -108,6 +108,11 @@ pub enum DigestItem<Hash> {
 	/// native code.
 	ChangesTrieSignal(ChangesTrieSignal),
 
+	/// Blockchain history digest item that contains the merkle mountain range root
+	/// at given block. It is created for providing super light client a commitment
+	/// of all the previous blocks.
+	MerkleMountainRangeRoot(Hash),
+
```

Comment:
This has be removed and deprecated, please do not base on this commit and version. choose and base on latest master branch.

Reply:
Thanks, I will rebase on latest master.